HW_25.10/HW_25.10/Decerialize.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System;
+using Microsoft.Extensions.Configuration;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using System.IO;
+using Newtonsoft.Json;
+
+namespace HW_25._10
+{
+    public class Decerialize : IDecerialize
```

Comment:
Call classes and interfaces like subjects, not verbs. So it would be better to use Decerializer and IDecerializer